<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogfournisseur</class>
 <widget class="QDialog" name="Dialogfournisseur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fournisseurs</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>831</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_11">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>601</x>
       <y>4</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Consulter Produits</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>571</x>
       <y>100</y>
       <width>221</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ajoutForm">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="f_cin">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>CIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="f_nom">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>NOM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="f_prenom">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>PRENOM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="f_tel">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>TELEPHONE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Date de naissance </string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="f_date">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="f_ajouterBtn">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>571</x>
       <y>440</y>
       <width>194</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="f_modifier">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="f_supprimer">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>-10</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gerer Fournisseurs</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>430</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="f_cin_e_1">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="f_nom_e_1">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="f_prenom_e">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="f_tel_e">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="f_daten_e"/>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="tableFournisseur">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>110</y>
       <width>511</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="f_recherche">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Recherche..</string>
     </property>
    </widget>
    <widget class="QPushButton" name="contactBtn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>480</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Contacter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrint">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>480</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTri">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CIN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NUMTEL</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="triBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>76</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>510</y>
       <width>491</width>
       <height>111</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>481</width>
         <height>86</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="status_adresse_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Adresse_ajout">
          <property name="text">
           <string>Ajout Adresse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="supprimer_Adresse">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>421</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox_VIlle">
          <item>
           <property name="text">
            <string>Ville</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tunis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sfax</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sousse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kairouan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bizerte</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gabes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ariana</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gafsa</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Monastir</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zaghouan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ben arous</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kasserine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mednine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mahdia</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SidiBouzid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Beja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jendouba</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Touzeur</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tataouine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gbelli</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ben Guerden</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>El Kef</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mannouba</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nabeul</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Rue"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Zip"/>
        </item>
        <item>
         <widget class="QPushButton" name="ajouter_Adresse">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Enregistrer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_12">
    <widget class="QPushButton" name="enoyerMail">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailTo">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>40</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailObjet">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>90</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="Mailbody">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>311</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
